libs-haskell/test-utils/DA/Test/Sandbox.hs

```diff
@@ -68,17 +69,16 @@ getSandboxProc SandboxConfig{..} portFile = do
                 ]
         else pure []
     pure $ proc sandbox $ concat
-        [ [ "--port=0", "--port-file", portFile ]
+        [ [ "--participant=participant-id=example,port=0,port-file=" <> portFile ]
         , tlsArgs
-        , [ timeArg ]
+        , Maybe.maybeToList timeArg
         , [ "--client-auth=" <> clientAuthArg auth | Just auth <- [mbClientAuth] ]
         , [ "--auth-jwt-hs256-unsafe=" <> secret | Just secret <- [mbSharedSecret] ]
-        , [ "--ledgerid=" <> ledgerId | Just ledgerId <- [mbLedgerId] ]
-        , dars
+        , [ "--ledger-id=" <> ledgerId | Just ledgerId <- [mbLedgerId] ]
```

Comment:
aren’t you missing the dar upload somewhere?

Reply:
yes, it is WIP by the newjoiner Andreas ;)

Reply:
oh i see, apologies